### PR TITLE
More codecleans

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Note weight3 in skinned primitives.
[Code of my plugin for blender](https://bitbucket.org/ShadowHunterRUS/bigworld-blender-tools-wot-wowp-wows/src/7fbf39618a49864f57069bd3c2bfd38c614c28a5/io_bigworld_model/import_bw_primitives.py?at=master&fileviewer=file-view-default#import_bw_primitives.py-203)

File: wot/ModelReader.py [hunk 1]
```diff
@@ -204,27 +211,28 @@ def readMaterial(self, element):
...
-				material.collisionFlags = item.text.strip()
+				material.collisionFlags = item_text
 			elif item.tag == 'materialKind':
-				material.materialKind = item.text.strip()
+				material.materialKind = item_text
 			elif item.tag == 'identifier':
-				material.ident = item.text.strip()
+				material.ident = item_text
 			elif item.tag == 'property':
-				if item.text.strip() == 'diffuseMap':
```
Comment (ghost) [inline wot/ModelReader.py:209]: Why '.tga' -> '.dds'?

Comment (SkaceKamen) [inline wot/ModelReader.py:209]: Because paths to textures are (were?) save as .tga, but actual textures are saved as .dds

Comment (ghost) [inline wot/ModelReader.py:209]: > were?

I don't know. I did not notice this.

Comment (SkaceKamen) [inline wot/ModelReader.py:209]: My guess is that they fixed it in _processed version. But I would still leave the replacement there for backwards compatiblity.

**Comment by SkaceKamen** [comment]
Thanks, the vt classes are nice touch :+1:

**Pull request merged**